src/main/resources/application.properties

```diff
@@ -107,63 +108,76 @@ provison.mail.sender=provision@opendevstack.org
+jenkinspipeline.quickstarter.be-scala-akka.repo=ods-quickstarters
 
-jenkinspipeline.quickstarter.beTypescriptExpress.desc=Backend - NodeJS/Express 
-jenkinspipeline.quickstarter.beTypescriptExpress.url=opendevstack/ods-quickstarters.git#production/be-typescript-express/Jenkinsfile
+jenkinspipeline.quickstarter.be-typescript-express.desc=Backend - NodeJS/Express 
+jenkinspipeline.quickstarter.be-typescript-express.repo=ods-quickstarters
 
 # Data Science quickstarter
-jenkinspipeline.quickstarter.dsJupyterNotebook.desc=Data Science - Jupyter Notebook
-jenkinspipeline.quickstarter.dsJupyterNotebook.url=opendevstack/ods-quickstarters.git#production/ds-jupyter-notebook/Jenkinsfile
+jenkinspipeline.quickstarter.ds-jupyter-notebook.desc=Data Science - Jupyter Notebook
+jenkinspipeline.quickstarter.ds-jupyter-notebook.repo=ods-quickstarters
 
-jenkinspipeline.quickstarter.dsRshiny.desc=Data Science - R-Shiny
-jenkinspipeline.quickstarter.dsRshiny.url=opendevstack/ods-quickstarters.git#production/ds-rshiny/Jenkinsfile
+jenkinspipeline.quickstarter.ds-rshiny.desc=Data Science - R-Shiny
+jenkinspipeline.quickstarter.ds-rshiny.repo=ods-quickstarters
 
-jenkinspipeline.quickstarter.dsMlService.desc=Data Science - Machine Learning
-jenkinspipeline.quickstarter.dsMlService.url=opendevstack/ods-quickstarters.git#production/ds-ml-service/Jenkinsfile
+jenkinspipeline.quickstarter.ds-ml-service.desc=Data Science - Machine Learning
+jenkinspipeline.quickstarter.ds-ml-service.repo=ods-quickstarters
 
 # Frontend Quickstarter
-jenkinspipeline.quickstarter.feAngular.desc=Frontend - Angular
-jenkinspipeline.quickstarter.feAngular.url=opendevstack/ods-quickstarters.git#production/fe-angular/Jenkinsfile
+jenkinspipeline.quickstarter.fe-angular.desc=Frontend - Angular
+jenkinspipeline.quickstarter.fe-angular.repo=ods-quickstarters
 
-jenkinspipeline.quickstarter.feReact.desc=Frontend - React
-jenkinspipeline.quickstarter.feReact.url=opendevstack/ods-quickstarters.git#production/fe-react/Jenkinsfile
+jenkinspipeline.quickstarter.fe-react.desc=Frontend - React
+jenkinspipeline.quickstarter.fe-react.repo=ods-quickstarters
 
-jenkinspipeline.quickstarter.feVue.desc=Frontend - Vue
-jenkinspipeline.quickstarter.feVue.url=opendevstack/ods-quickstarters.git#production/fe-vue/Jenkinsfile
+jenkinspipeline.quickstarter.fe-vue.desc=Frontend - Vue
+jenkinspipeline.quickstarter.fe-vue.repo=ods-quickstarters
 
-jenkinspipeline.quickstarter.feIonic.desc=Mobile - Ionic
-jenkinspipeline.quickstarter.feIonic.url=opendevstack/ods-quickstarters.git#production/fe-ionic/Jenkinsfile
+jenkinspipeline.quickstarter.fe-ionic.desc=Mobile - Ionic
+jenkinspipeline.quickstarter.fe-ionic.repo=ods-quickstarters
 
 #Other Quickstarter
-jenkinspipeline.quickstarter.beAirflow.desc=Other - Airflow
-jenkinspipeline.quickstarter.beAirflow.url=opendevstack/ods-quickstarters.git#production/be-airflow/Jenkinsfile
+jenkinspipeline.quickstarter.docker-plain.desc=Docker/Plain
```

Comment:
Please change description to _Other - Docker/Plain_, because Quickstarters are sorted by description.